src/universal/components/TeamInvitationGoogleCreateAccount.tsx

```diff
@@ -99,21 +100,32 @@ class TeamInvitationGoogleCreateAccount extends Component<Props, State> {
   render () {
     const {error, submitting, verifiedInvitation} = this.props
     const {isEmailFallback} = this.state
-    const {teamInvitation, teamName} = verifiedInvitation
+    const {meetingType, teamInvitation, teamName} = verifiedInvitation
     if (!teamInvitation) return null
     const {email} = teamInvitation
     return (
       <StyledDialog>
         <Helmet title={`Sign up with Google | Team Invitation`} />
-        <DialogTitle>Welcome!</DialogTitle>
+        <DialogTitle>
+          {meetingType
+            ? `Join ${meetingTypeToLabel[meetingType]} Meeting in Progress`
```

Comment:
title is too long, consider `Join Meeting`

![Screenshot from 2019-06-18 07-34-39](https://user-images.githubusercontent.com/5514175/59693340-98248780-919b-11e9-8167-030eb3063ad1.png)